source/help/messaging/formatting-text.rst

```diff
@@ -163,46 +183,81 @@ Renders as:
 
 
 In-line Code
-============
+------------
 
 Create in-line monospaced font by surrounding it with backticks.
 
-.. code::
+.. code-block:: none
+
 
   `monospace`
 
 Renders as: ``monospace``.
 
 In-line Images
-==============
+--------------
 
-Create in-line images using an ``!`` followed by the alt text in square brackets and the link in normal brackets. Add hover text by placing it in quotes after the link.
+Add in-line images by using an exclamation mark followed by the alt text in square brackets then the URL of the image in round brackets. You can add hover text by placing the text in quotes after the link.
 
-.. code::
+**![** *{alt text}* **](** *{URL of image}*  "*{Hover text}* " **)**
 
-  ![alt text that shows when a link is broken](broken-link "hover text")
+If the height of the original image is more than 500 pixels, the system sets the height at 500 pixels and adjusts the width to maintain the original aspect ratio.
 
-  and
+You can set the width and height of the displayed image by using an equals sign and the width x height after the URL of the image. If you set only the width, the system adjusts the height to maintain the original aspect ratio.
 
-  [![Build Status](https://travis-ci.org/mattermost/platform.svg?branch=master)](https://travis-ci.org/mattermost/platform) [![Github](https://assets-cdn.github.com/favicon.ico)](https://github.com/mattermost/platform)
 
-Renders as:
+**Examples**:
+
+Inline image with hover text
+  .. code-block:: none
+
+    ![Github](https://assets-cdn.github.com/favicon.ico "GitHub Icon")
+
+  Renders as:
+
+  .. raw:: html
 
-.. image:: https://docs.mattermost.com
-  :alt: alt text that shows when a link is broken
+    <img src="https://assets-cdn.github.com/favicon.ico" alt="GitHub" title="GitHub Icon"></a>
```

Comment:
Should we be linking to images on Github's servers? It doesn't seem likely that this link will ever break, but it seems like it would be good practice to only link to images we control